src/bodywork/exceptions.py

```diff
@@ -38,19 +38,23 @@ def __init__(self, failed_jobs: Iterable[V1Job]):
         super().__init__(msg)
 
 
-class BodyworkConfigParsingError(Exception):
+class BodyworkConfigError(Exception):
```

Comment:
I'm not sure why this exists - it's acting as a wrapper for `Exception`. Superfluous, no?

Reply:
It's a base exception class for Config errors so that when we check the exception type in run_workflow we don't need to check each one so its a lot cleaner.

Reply:
Understood - good idea.